features/send-money-to-receiver.feature

```diff
@@ -24,6 +24,7 @@ Feature: Send Money to Receiver
       | balance  |
       | 0.000010 |
 
+  @watch
```

Comment:
Please remove

Reply:
Thanks @DominikGuzei . I will do fixes you mentioned 😉 . Tests failed only in one case which @nikolaglumac mention above.